src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -137,7 +137,11 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
 
     // Replaces tooltip text of the {@code button}.
     const tooltipString = $(button).attr('data-original-title').replace(from, to);
-    $(button).attr('title', tooltipString).tooltip('fixTitle').tooltip('show');
+    $(button).attr('title', tooltipString).tooltip('fixTitle');
+}
+
+function isInExpandMode(button) {
+    return $(button).html().trim().startsWith('Expand');
 }
```

Comment:
I meant we can move this function inside the other function so we can continue to use the `STRING_EXPAND` constant.